README.md

```diff
@@ -1,19 +1,218 @@
+  and block any PR in the repository from being merged.
+- `email`: Allstar could send an email somehow.
+- `rpc`: Allstar could send an rpc to some organization-specific system.
+
+## **Policies**
+
+Similar to the Allstar app enable configuration, all policies are enabled and
+configured with a yaml file in either the organization's `.allstar` repository,
+or the repository's `.allstar` directory. As with the app, policies are opt-in
+by default, also the default `log` action won't produce visible results. A
+simple way to enable all policies is to create a yaml file for each policy with
+the contents:
+
+```
+optConfig:
+  optOutStrategy: true
+action: issue
+```
+
+
+### Branch Protection
+
+This policy's config file is named `branch_protection.yaml`, and the [config
+definitions are
+here](https://pkg.go.dev/github.com/ossf/allstar@v0.0.0-20210728182754-005854d69ba7/pkg/policies/branch#OrgConfig).
+
+The branch protection policy checks that GitHub's [branch protection
+settings](https://docs.github.com/en/github/administering-a-repository/defining-the-mergeability-of-pull-requests/about-protected-branches)
+are setup correctly according to the specified configuration. The `fix` action
+is not yet implemented, but should be possible and is planned.
+
+### Binary Artifacts
+
+This policy's config file is named `binary_artifacts.yaml`, and the [config
+definitions are
+here](https://pkg.go.dev/github.com/ossf/allstar@v0.0.0-20210728182754-005854d69ba7/pkg/policies/binary#OrgConfig).
+
+This policy incorporates the [check from scorecard](https://github.com/ossf/scorecard/#scorecard-checks). The `fix` action is not planned.
+
+### Outside Collaborators
+
+This policy's config file is named `outside.yaml`, and the [config definitions
+are
+here](https://pkg.go.dev/github.com/ossf/allstar@v0.0.0-20210728182754-005854d69ba7/pkg/policies/outside#OrgConfig).
+
+This policy checks if any [Outside
+Collaborators](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/adding-outside-collaborators-to-repositories-in-your-organization)
+have either administrator(default) or push(optional) access to the
+repository. Typically only organization members have this access.
+
+The `fix` action is not implemented. I suppose Allstar could remove the access,
+we need to check the permissinos we are currently requesting to see if this is
+possible.
+
+### SECURITY.md
+
+This policy's config file is named `security.yaml`, and the [config definitions
+are
+here](https://pkg.go.dev/github.com/ossf/allstar@v0.0.0-20210728182754-005854d69ba7/pkg/policies/security#OrgConfig).
+
+This policy checks that the repository has a security policy file in `SECURITY.md` and that it is not empty.
+
+The `fix` action is not implemented. Possibly look into how the `.github`
+organizaiton level repository works, and/or see how to open a PR with a set
```

Comment:
typos and confusing recommendation, are we putting developement instruction in docs, can exclude those here and instead put these in github issues for these features.

Reply:
removed.